packages/icons/package.json

```diff
@@ -9,9 +9,6 @@
     "start": "snowpack dev",
     "build": "node --es-module-specifier-resolution=node ./dist/utils/ssr.js"
   },
-  "dependencies": {
-    "@masknet/theme": "workspace:*"
```

Comment:
If you remove this, please also remove it from tsconfig file.

Reply:
I don't see any reference to `@masknet/theme` in `packages/icon/tsconfig.json`. What I miss?

Reply:
Oh so that was a mistake when adding that dependency